components/Icons.tsx

```diff
@@ -190,10 +190,15 @@ export function ArrowRight(props: { class?: string }) {
   );
 }
 
-export function TriangleRight(props: { class?: string }) {
+export function TriangleRight(
+  props: { class?: string; tabindex?: number; onKeyDown?: string },
+) {
   return (
     <svg
       class={tw(props.class ?? "")}
+      tabindex={props?.tabindex}
+      // @ts-ignore onKeyDown does support strings
```

Comment:
Is it necessary to use a string though?

Reply:
Yes, else if we'd use a normal callback, it would have to be an island